### ISSUE TITLE
Support ANSI lumens as a measure of light output

### ISSUE THREAD

**Issue by gstorhaug** [Ocupe/Projectors#15]
The amount of light per watt is very different for a LED/laser projector versus a traditional UHP projector.  Specifying the light output as ANSI lumens would give much more predictable/true to life results.

**Comment by Ocupe** [comment]
@gstorhaug thanks for opening the issue. If I remember correctly I went with watt because it's the unit Blender uses to handle lights [[Link to Docs](https://docs.blender.org/manual/en/latest/render/lights/light_object.html#power-of-lights)]. 

> If you want to set the power to real world values, you have to convert the wattage of consumer bulbs or LED lights to radiant flux, but it is not a straightforward process.

It looks like a 1 to 1 convertions is not so simple. Do you have any resources or ideas to implement this?

Another open question: does the brightness change when I change the throw ratio (while the projector remains in the same place)? I need to check this, but I don't think I implemented it that way 🤔

So the way I handle the light output is not so physics based after all :D
I will look into it, but feel free to give me your thoughts and ideas.

**Comment by gstorhaug** [comment]
Yes, I noticed (after adding the issue) that Blender uses watt as a unit for lights (which frankly makes no sense).  There are tables for converting between lumens and energy consumption (watts), e.g. https://en.wikipedia.org/wiki/Lumen_(unit)#Lighting.  The lumens in this table is however not the same as ANSI Lumens.  ANSI Lumens was created as a standardized way of measuring the light output from projectors, to make it easier to compare different projectors.

One advantage of using ANSI Lumens is that it is then easy to compute the illuminance to get a feeling for how light the projected image will be (illuminance = luminous flux per unit area).  I have this ability in an inhouse developed system, and find it very useful when planning multi-projector systems.  (I am however not allowed to share our inhouse developed system).

Maybe the easiest thing to do is to ignore this feature request, at least until Blender uses lumens (instead of watts) to set the brightness of a light source.
